_includes/data_download.html

```diff
@@ -1,10 +1,10 @@
       {% if page.url == '/data/' %}
         {% capture page_title %}  - Data{% endcapture %}
-        {% assign data_url = site.data_url %}
+        {% assign data_prefix = 'live' %}
```

Comment:
Same as above, the `/live` prefix will be used for data download pages when the URL is `/data/`. Crude, but works.